src/platforms/windows/daemon/wireguardutilswindows.h

```diff
@@ -52,7 +50,7 @@ class WireguardUtilsWindows final : public WireguardUtils {
   std::unique_ptr<WireGuardAPI> m_wireguard_api;
   ulong m_deviceIpv4_Handle = 0;
 
-  WindowsRouteMonitor m_routeMonitor;
+  WindowsRouteMonitor* m_routeMonitor = nullptr;
```

Comment:
Nitpick, maybe a QPointer, so that 
```
   delete m_routeMonitor;
    m_routeMonitor = nullptr;
``` 

could be condensed to 
`m_routeMonitor->deleteLater()`